### ISSUE TITLE
When using Custom URLs for tiles, the background type menu refer to it as "Aerial (by ESRI)"

### ISSUE THREAD

**Issue by govvin** [Helium314/SCEE#506]
When a custom URL is set as a tile source, the option for selecting a background type still refers to it as "Aerial (by ESRI)". Perhaps we should have a separate entry for default aerial layer, then something else for custom layer?

Not a show-stopper, but a consistent interface for any app is always nice to have. Thanks for the awesome work on SCEE!

**How to Reproduce**
Add a custom tile layer URL from Settings > SCEE settings > Data Management > Tile source for satellite/aerial images.

**Expected Behavior**
The menu option will show "custom URL" (or something similar) instead of "Aerial (by ESRI)

**Does it happen in normal StreetComplete?**
I don't use it, but as far as I can tell this option is only available in SCEE?

**Versions affected**
v56.0

**Screenshot**
Background type selection options, showing "Aerial (by ESRI), even if a custom URL has been set.

<img src="https://github.com/Helium314/SCEE/assets/5821688/4cd5afef-2569-46f5-b6db-c1ba72c7506d" width="320" />

**Comment by Helium314** [comment]
It would be nice to have, but certainly is not priority.

Considering that I'm not sure whether (and if so, when) I'll re-implement aerial imagery access after switching to mapLibre (https://github.com/streetcomplete/StreetComplete/issues/3123), I will not work on this in forseeable future.

**Comment by mnalis** [comment]
That particular menu entry is not there, but some things (split way, insert node etc) still use that `R.string.background_type_aerial_esri = Aerial (by Esri)` 

Perhaps just calling it `Aerial` without further `(by Esri)` qualifications would suffice for now? 

At least if/until we got possibility of multiple background maps (PR https://github.com/Helium314/SCEE/pull/564); when those would need to be reworked anyway

**Comment by Helium314** [comment]
Calling it "Aerial" might also be incorrect, as we don't know what users do here. I remember someone was using Carto tiles.

So only "correct" thing would be using something like "Custom" if it's not the default raster background.

**Comment by mnalis** [comment]
> Calling it "Aerial" might also be incorrect, as we don't know what users do here

That is true, but I do not feel being extra precise there is of such huge importance in SCEE. Perhaps just calling it _"Raster"_ would suffice in current case (as default _"Map"_ is vector-based)?

**Comment by mnalis** [comment]
> So only "correct" thing would be using something like "Custom" if it's not the default raster background.

If there is really the need to dynamically change the name of the button depending on what background is being selected, then perhaps using (trimming the TLD) domain name as  a name of the button would work? e.g. with default raster URL it would be called `arcgisonline`, and if I change the URL to point somewhere on dgu.hr domain the button would say `dgu`. 

But I don't really see amount of work needed to be done being proportional to the usefulness for current version of SCEE... 

It will however make more sense for the effort if  https://github.com/Helium314/SCEE/pull/564 (implementing **_multiple_** background URLs) gets merged, as then we'd need to distinguish and switch from `raster#1` to `raster#2`.

**Comment by andrewharvey** [comment]
Strange in my SCEE, under Settings Display there is no Select Background type option, instead under Data Management there is "Tile source for satellite / aerial images". No mention of ESRI. Perhaps this issue is fixed already?

**Comment by govvin** [comment]
Issue already fixed.

**Comment by mnalis** [comment]
> No mention of ESRI. Perhaps this issue is fixed already?

Note that in some cases (e.g. `It's at the slighly different position...` other answer), there is still a button named `Aerial (by Esri)` that allow switching map view while moving the node.

**Comment by mnalis** [comment]
- Well, that menu seems back in SCEE v60.2 (under `Settings` / `Display` / `Select background type`, so reopening:

  <details>
  <summary>screenshot</summary>

  ![Image](https://github.com/user-attachments/assets/a78f1945-9f11-4914-a961-194a9b05255e)

  </details>

- also  in some cases (e.g. `It's at the slighly different position...` other answer), there is still a button named `Aerial (by Esri)` that allow switching map view while moving the node.
